### ISSUE TITLE
Can't log in

### ISSUE THREAD

**Issue by f1ps1** [lukechilds/dockerpi#13]
After I download and start the image `docker run lukechilds/dockerpi` I get prompted to enter username/password.

After I enter `pi` the container freezes.

I already waited for a few minutes because I thought the SSH keygen takes so long but it still didn't respond after +5mins.

After I press ctrl+c it shows me `qemu-system-arm: terminating on signal 2` sometimes also `qemu-system-arm: terminating on signal 2`

**Comment by lukechilds** [comment]
What are the specs of your host machine?

It can be a bit slow, although I haven't seen it hang for that long.

**Comment by f1ps1** [comment]
I'm running docker in an Ubuntu vm. This shouldn't be a problem because nested virtualization is enabled and docker already worked with other containers.

My Windows host specs are: 8 GB RAM, i5 7th gen, Windows 10 Home 64-bit
My vm specs: 4 GM ram, 4 cores, Ubuntu MinimalCD 18.04 64-bit

**Comment by lukechilds** [comment]
Do you have Docker for Windows installed on the host? Does it work there?

The QEMU virtualisation for the Pi machine is virtualising ARM so it won't be able to take advantage of any hardware acceleration.

**Comment by f1ps1** [comment]
No, I can't install docker on the Windows host, because docker works only for Windows Enterprise and Professional but I have Windows Home.

I'll try again and wait for the log in to finish and let you know of the results

**Comment by opensourcemukul** [comment]
username: pi
password: raspberry
